composer.json

```diff
@@ -103,5 +103,12 @@
         "psr-0": {
             "Sulu\\": "tests/"
         }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-release/1.6": "1.6.x-dev",
+            "dev-release/2.0": "2.0.x-dev",
```

Comment:
We could skip 1.6 and 2.0 branch aliases if we remove the `release/` prefix from the branches see: https://getcomposer.org/doc/articles/aliases.md#branch-alias

Reply:
would prefer to change our branch names if that is what composer expects per default

Reply:
In that case I would also prefer to change the branch names, but that operation is always a bit risky, because PRs might get closed, URLs might break (or does github create correct redirects for them?), ...

Apart from that we have to adjust these lines always when a new version is released, because at least the `master` branch has to be kept up do date... So I think I would currently not take the risk.